contracts/contracts/listings/v01/Listings.sol

```diff
@@ -0,0 +1,206 @@
+pragma solidity 0.4.23;
+
+import "../../EvolvingRegistry.sol";
+import "./Escrow.sol";
+
+contract V01_Listings {
+  event ListingChange(uint256 indexed _listingIndex, bytes32 _ipfsHash);
```

Comment:
The `_listingIndex` in this event is the same as the `_entryIndex` in the registry events. Perhaps since are the same number, they should be called by the same name.